src/bot/routes/rooms.rs

```diff
@@ -199,3 +241,21 @@ async fn get_by_id_from_matrix(id: &str, matrix_client: &MatrixClient) -> Result
     Ok(community)
 }
 
+
+fn name_to_slug(name: &str) -> String {
+    let filtered: String = name.chars().filter_map(|c| {
+        match c {
+            ' ' => Some('_'),
+            _ if c.is_alphabetic() => Some(c),
```

Comment:
`is_ascii_alphanumeric` seems a better option. ascii to check each char is 1 byte and alphanumeric to allow numbers.